projects/museumpassmusees/docs/introduction.md

```diff
@@ -2,32 +2,14 @@
 
 ![](../assets/images/logo.png)
 
-Welcome to the museumPASSmusées API documentation!
+## What is museumPASSmusées?
 
-## Partner API
-
-With the museumPASSmusées Partner API you can look up the card of a passholder and check if they are allowed to visit your museum and register their visit so your museum gets reimbursed for it. *(Coming soon)*
-
-It also enables you to sell museumPASSmusées subscriptions to new passholders by generating voucher codes and selling those to new passholders to use in the online order process on [museumpassmusees.be](https://museumpassmusees.be).
-
-## Authentication
-
-To use the museumPASSmusées APIs as they are documented here, you will need a **client id** and **client secret** to retrieve a [client access token](https://docs.publiq.be/docs/authentication/4706c3721316a-client-access-token) to authenticate your requests.
-
-To call an endpoint, your client also needs specific permissions linked to a role.
+With museumPASSmusées, Belgian museum visitors enjoy access to over 200 museums for a small yearly price.
```

Comment:
@DizzyMissLizzy I couldn't find a good summary of what museumPASSmusées is (especially in English) so I wrote this myself. But maybe you can write a better introduction? I think it would especially be good to focus on the museum's perspective instead of the passholder's perspective here, since the API is meant to be used by museums.